tests/identity/id.test.ts

```diff
@@ -129,9 +128,12 @@ describe('identity', () => {
     );
 
     const dbProfile = await dataStorage.identity.getProfileByVerifier('http://polygonissuer.com/');
-    expect(dbProfile.id).to.equal(profileDID.string());
-    expect(dbProfile.genesisIdentifier).to.equal(did.string());
-    expect(dbProfile.nonce).to.equal(10);
+    expect(dbProfile).not.to.be.undefined;
```

Comment:
Why do you need this check? If dbProfile is undefined the next tests will be failed.

Reply:
linter. there is a check for non-undefined